examples/meson.build

```diff
@@ -47,6 +47,7 @@ if libavutil.found() and libavcodec.found() and libavformat.found()
 	executable(
 		'dmabuf-capture',
 		'dmabuf-capture.c',
-		dependencies: [wayland_client, wlr_protos, libavutil, libavcodec, libavformat, wlroots]
+		dependencies: [wayland_client, wlr_protos, libavutil, libavcodec, libavformat, wlroots],
+		link_args : '-pthread',
```

Comment:
Meson has a proper way you're supposed to link against pthreads: http://mesonbuild.com/Threads.html